nncf/torch/model_analyzer.py

```diff
@@ -81,7 +81,4 @@ def is_quantized_weights(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
     :param nncf_graph: The NNCF graph.
     :return bool: return `True` if the node is quantized.
     """
-    for prev_node in nncf_graph.get_previous_nodes(node):
-        if prev_node.node_type in OP_NAMES_QUANTIZE_NODE:
-            return True
-    return False
+    return any(prev_node.node_type in OP_NAMES_QUANTIZE_NODE for prev_node in nncf_graph.get_previous_nodes(node))
```

Comment:
Well, the previous version looks slightly better than this, even with unapplicable rules.

Reply:
Yes, that's debatable, for me `any` looks better and shorter. But ok, disabled this rule and reverted changes.